pom.xml

```diff
@@ -49,7 +49,7 @@
   <parent>
     <groupId>org.jahia.modules</groupId>
     <artifactId>jahia-modules</artifactId>
-    <version>8.1.0.0</version>
+    <version>8.1.3.0-SNAPSHOT</version>
```

Comment:
just checking, why do we need the bump here? I don't know if it's necessary to add to registry.

Reply:
Don't want to break registry and also before release those changes won't have any impact so useless to deploy.
If there is other changes coming we will advise